sceptre/config/reader.py

```diff
@@ -467,8 +478,11 @@ def _render(self, directory_path, basename, stack_group_config):
             try:
                 config = yaml.safe_load(rendered_template)
             except Exception as err:
+                debug_file_path = self._write_debug_file(rendered_template)
+
                 raise ValueError(
-                    "Error parsing {}:\n{}".format(abs_directory_path, err)
+                    f"Error parsing {abs_directory_path}:\n{err}\n"
+                    f"Rendered template saved to: {debug_file_path}"
```

Comment:
if the user is processing a config group does sceptre exit on the 1st failure or does it continue to process all templates thus printing this error message multiple times? if it's the latter then we might want to capture all errors.

Reply:
I think [my suggestion above](https://github.com/Sceptre/sceptre/pull/1321#discussion_r1166827324) would simplify this.

Reply:
@zaro0508 An exception is raised so it exits on the first failure.